PennMobile/src/main/java/com/pennapps/labs/pennmobile/MainActivity.java

```diff
@@ -118,6 +124,11 @@ public void onBackPressed() {
     public void onResume() {
         super.onResume();
         setTitle(R.string.main_title);
+        NotificationManager manager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        manager.cancelAll();
+        if(from_alarm){
```

Comment:
Style: there should be a space on both sides of the condition, so `if (from_alarm) {`